js/data/locale/tg/unitfmt.json

```diff
@@ -8,10 +8,10 @@
             "degree": "#{n}°",
             "arc-minute": "#{n}′",
             "arc-second": "#{n}″",
-            "square-kilometer": "#{n} km²",
+            "square-kilometer": "#{n} километри мураббаъ",
             "hectare": "#{n} ha",
             "square-meter": "#{n} m²",
-            "square-centimeter": "#{n} cm²",
+            "square-centimeter": "#{n} сантиметр мураббаъ",
             "square-mile": "#{n} mi²",
             "acre": "#{n} ac",
```

Comment:
Need the longer forms for the other measures as well... the most important ones first, like "meter" and "kilometer"